### DISCUSSION TITLE
What are the bounce limits? (Not self hosted)

### DISCUSSION THREAD

**Discussion by ledgeri** [simple-login/app#830] [Q&A]
Hi!

I have started using the service in the last month heavily, migrating my accounts (i mean other accounts, changing the email adresses to SL alias). I did it in a way, when i still tunneling emails to my main email adress (meaning the content of the emails do not really differ), but in an order, as a basement for future changes. I am still withn the free-account alias-limit with 100+ accounts, so i have alias overlaps. (E.g. why bother giving individual alias to all of my steam games, what using an individual account.. etc)
The migration was the usual, log in, verify email adress if needed, change password, change email, verify new email adress, done.
(I know the bounce decided only on the Email-provider's side, my issue is on the SL side)

During the time of migration i experienced 2 kind of bounced-mail behaviour.
--In the early phase i got a bounce here, and there, but usually the second email gone through. I have recived an individual notification if an individual bounce happened, checked the email in a standalone client, and the confirmation of the other-account's change usually arrived. So if there was a bounce, got a notification mail from that specific bounce.
--The 2nd phase started about a week ago: if a bounce occurs, i do not recive a notification about that bounce. Instead i recive a notification about the used alias being disabled because "several emails bounced". I not have to mention, why it is worse.
Addational info: 
I even experienced phase 2 on an alias, what was not recived any bounce before, but only in phase 2.
Once it happened in font of me: i turned back online my alias, in 5 minute it was disabled again.

So my related questions: 
What happened that caused the bounce behaviour change? Is there any way to change it back?
If i have reached a limit, what is that limit? Timaframe based, total account bounce based?
Why attacking the alias, is better than attacking the sender on the system level? Why there is no option to choose about this one in the preferences?

**Comment by nguyenkims** [comment]
We have made some changes in the anti-abuse detection which is responsible for automatically disabling an alias if it turns "rogue". The detection should be much more tolerable now and the possibility of a false positive is now unlikely.

> Why attacking the alias, is better than attacking the sender on the system level? Why there is no option to choose about this one in the preferences?

We already have some protections on the sender level. However an attacker can easily change the sender (envelope and header) which makes this detection more difficult.

**Reply by ledgeri** [reply]
> We have made some changes in the anti-abuse detection which is responsible for automatically disabling an alias if it turns "rogue".

Am i felt the time-frame right? Is thit happened around a week ago?

> The detection should be much more tolerable now and the possibility of a false positive is now unlikely.

As we speek, or as the mentioned rouge-detection changes happened earlier?


> We already have some protections on the sender level. However an attacker can easily change the sender (envelope and header) which makes this detection more difficult.

Understandable... but does the detecting algorithm counts in SL-user actions, like turning back autoblocked aliases, or something similar?

I am asking these because if i change email provider, and if that -for some reson- bounces more emails than my current (what i was unable to check) there is a chance to my outsider-accounts stuck on aliases -because constant alias blockage-, or i have to revert to my previous provider to change outsider-account emails from SL-aliases- and by that render simple login meaningless to me... The previous system at least gave the chance to check if one type of sender's gets blocked a lot, so that could be deattachable, without risking the others on that alias. Now, if a valid sender's mail bouncing, can lead to a "confirmation email not arrived" loophole.

Thanks for the replies!

**Reply by nguyenkims** [reply]
We realized that the algorithm was too sensible a week ago and made a change yesterday to make it more tolerable and less sensible to fault positives.

> Understandable... but does the detecting algorithm counts in SL-user actions, like turning back autoblocked aliases, or something similar?

As the auto-blocking is based on bounces and if an alias is blocked, it doesn't forward emails, we don't have a way to auto-enable an alias.

Usually bounces only happen when there's an issue with a mailbox provider. We are actively monitoring and whenever this happens, get in touch with the mailbox provider to solve the issue asap. With the previous algorithm, during the resolution time, bounces can continue to happen that might lead to  aliases being disabled. With the new algorithm, this shouldn't happen.

**Reply by ledgeri** [reply]
Thank you for the information!

> As the auto-blocking is based on bounces and if an alias is blocked, it doesn't forward emails, we don't have a way to auto-enable an alias.
That is sure thing, but what i meant is: if an auto lock happened, because of the initilally predefined events, does the algorithm able to detect if the user goes to that alias and turns it back? Does this act has any cause/effect on future behaviour, even if for a limited time?
